include/oneapi/internal/daal_level_zero_common.h

```diff
@@ -17,7 +17,7 @@
 
 #ifndef _DAAL_LEVEL_ZERO_COMMON
 
-    #ifndef __linux__
+    #if !(defined(__linux__) || defined(_WIN32) || defined(_WIN64))
```

Comment:
Does L0 support WIN32? As far as I know, even the DPCPP does not support 32 bit systems.

Reply:
ok, removed defined(_WIN32) from condition